CSharp/BatchExplorer/Models/JobModel.cs

```diff
@@ -74,6 +76,9 @@ public bool IsChecked
         /// </summary>
         [ChangeTracked(ModelRefreshType.Children)]
         public ICollectionView TaskCollection { get; private set; }
+
+
+        public string OutputStorageContainerName { get { return this.Job.OutputStorageContainerName(); } }
```

Comment:
Missing documentation comment - and does this need a `[ChangedTracked]` attribute?